components/Form.js

```diff
@@ -0,0 +1,73 @@
+import { useRouter } from "next/router";
+import { styled } from "styled-components";
+
+export default function Form({ submitNewCard, formName }) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    submitNewCard(data.name, data.icon);
+
+    event.target.reset();
+    router.push("/");
+  }
+
+  return (
+    <StyledForm aria-labelledby={formName} onSubmit={handleSubmit}>
+      <label htmlFor="name">Name or Text:</label>
+      <input
+        id="name"
+        name="name"
+        type="text"
+        placeholder="Text for your card"
+        pattern="[a-zA-Z]*"
+        required
+      />
+      <label htmlFor="icon">Choose an icon:</label>
+      <select name="icon" id="icon">
+        <option value="">--Please choose an icon--</option>
+        <option value="🚜">🚜</option>
+        <option value="🐴">🐴</option>
+        <option value="🦌">🦌</option>
+        <option value="🌻">🌻</option>
+        <option value="🌈">🌈</option>
+        <option value="🏍️">🏍️</option>
+        <option value="🚐">🚐</option>
+        <option value="🚚">🚚</option>
+        <option value="🚍">🚍</option>
+        <option value="🚲">🚲</option>
+        <option value="🛴">🛴</option>
+        <option value="🚦">🚦</option>
+        <option value="⛽️">⛽️</option>
+        <option value="🌅">🌅</option>
+        <option value="⛰️">⛰️</option>
+        <option value="🔥">🔥</option>
+        <option value="⛺️">⛺️</option>
```

Comment:
Would you mind changing these values to the corresponding text value? Such as `<option value="rainbow">🌈</option>`.

Reply:
I have used the icons as value because otherwise ther text is output to me. I plan to get these icons via an API. If it is ok, I would leave this so.